### PR TITLE
Remove API endpoints equivalence table from `Migrating from the Wazuh API 3.X`

### PR DESCRIPTION
**Pull request by mcarmona99**
## Description

This PR closes https://github.com/wazuh/wazuh-documentation/issues/5210.

In this pull request, I have removed the equivalence table from the `Migrating from the Wazuh API 3.X` section.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by vicferpoy — CHANGES_REQUESTED** [review]
Missing changes on `redirect.js`.

> **EDIT:** It was not necessary. The page will remain.

**Review by vicferpoy — APPROVED** [review]
LGTM

File: source/user-manual/api/equivalence.rst [hunk 1]
```diff
@@ -26,8 +26,8 @@ Wazuh API v4 introduces several new endpoints and also modifies or removes some
 Migrating users
 -----------------
 
-The Wazuh API users are not migrated when upgrading the Wazuh API from 3.X to 4.3. It is not recommended to migrate the users since there are numerous security changes to the Wazuh API.
-However, using the following API calls it is easy to create new users and assign the administrator role to it (substitute <username> and <password>):
+The Wazuh API users are not migrated when upgrading the Wazuh API from 3.X to 4.X and newer versions. It is not recommended to migrate the users since there are numerous security changes to the Wazuh API.
+However, it is easy to create new users and assign the administrator role to them with the following API requests (substitute <username> and <password>):
```
Comment (s-ocando) [inline source/user-manual/api/equivalence.rst:30]:
```suggestion
However, it is easy to create new users and assign them the administrator role with the following API requests (substitute <username> and <password>):
```

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**